install/0000_30_machine-api-operator_02_machine.crd.yaml

```diff
@@ -27,6 +27,10 @@ spec:
     description: Zone associated with machine
     name: Zone
     type: string
+  - JSONPath: .status.nodeRef.name
```

Comment:
It needs to be set in `machine_types.go` first and the CRD updated by running `make gen-crd`. Otherwise, your  change will get removed after the CRD gets regen again. Let's collect all the bits in https://github.com/openshift/cluster-api/pull/55 first.